katello-attach-subscription

```diff
@@ -102,11 +106,115 @@ end
   end
 end
 
+# check functions
+def checksubs()
+  # initialize variables
+  # by default any kind of subscription is fine
+  subfiltertype = nil
+
+  # create api binding
+  api = ApipieBindings::API.new({:uri => @options[:uri], :username => @options[:user], :password => @options[:pass], :api_version => '2', :timeout => @options[:timeout]})
+
+  # If type is Hypervisor, only NORMAL subscription has to be considered
+  if @yaml[:type] = "Hypervisor"
```

Comment:
Where does that come from? We don't define `type` directly in the YAML, just in the individual subscription sections